content/altinn-studio/v9/develop-a-service/expressions/reference/functions/_index.nb.md

```
@@ -56,6 +56,10 @@ Disse funksjonene er tilgjengelige for bruk i uttrykk:
 | [`displayValue`](#func-displayValue)      | Streng                           | Streng     | ✅       | ❌      |
 | [`round`](#func-round)                    | Tall, valgfritt Tall             | Streng     | ✅       | ✅      |
 | [`formatDate`](#func-formatDate)          | Dato/tid, valgfri Streng         | Streng     | ✅       | ✅      |
+| [`plus`](#func-plus)                      | Ett eller flere tall             | Tall       | ✅        | ✅      |
+| [`minus`](#func-minus)                    | Tall], tall                      | Tall       | ✅        | ✅      |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)